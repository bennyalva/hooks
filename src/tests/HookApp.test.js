import '@testing-library/jest-dom';
import { HooksApp } from '../HooksApp';
import { shallow } from 'enzyme';
import React from 'react';

describe('HookAppTest', () => {
   test('should to beCorrectly HookApp', () => {

        const wrapper = shallow(<HooksApp />);
        expect(wrapper).toMatchSnapshot();
       
   });
    
});
