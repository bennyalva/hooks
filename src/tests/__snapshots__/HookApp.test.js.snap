// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HookAppTest should to beCorrectly HookApp 1`] = `
<div>
  <h1>
    Hola mundo
  </h1>
</div>
`;
